README.md

```diff
@@ -31,32 +31,6 @@ XCode 9.0
 
 ## Building from Source
 
```

Comment:
Add link to Opencv readme. :)
Other LGTM

Reply:
updated